NEWS.rst

```diff
@@ -0,0 +1,130 @@
+
+0.7.0 (2014-01-15):
+  Major:
+
+  - Only *Python* 2.6 and 2.7 are supported now. 2.4, 2.5 is deprecated.
+  - setup.py now uses setuptools, and stores its dependencies. This
+    means you no longer need to manually install dependencies.
+  - *setup.py* no longer requires *Twisted* for *egg_info*, making it easier
+    to install *Epsilon* using *pip*.
+  - Significant improvements to *PyPy* support. *PyPy* is now a supported
+    platform, with CI support.
+  - ``epsilon.release`` is now removed. It relied on a bunch of machinery
+    specific to *Divmod* that no longer existed.
+  - ``epsilon.sslverify`` is now removed. Use ``twisted.internet.ssl`` instead.
+  - ``epsilon.asTwistedVersion`` takes a string version (``"1.2.3"``) and
+    turns it into a ``twisted.python.versions.Version``.
+
+  Minor:
+
+  - Several deprecation warnings have been cleaned up.
+
+0.6.0 (2009-11-25):
+  - Disable loopback hotfix on *Twisted* 8.2 and newer.
+  - Remove the implementation of ``Cooperator`` and use *Twisted*'s
+    implementation instead.
+  - Use *Twisted*'s ``deferLater`` implementation.
+  - Add a service for communicating via *stdio*.
+  - Add a ``precision`` argument to ``Time.asHumanly`` to control the precision
+    of the returned string.
+
+0.5.12 (2008-12-09):
+  - Added support for *AMP* authentication via one-time pads.
+
+0.5.11 (2008-10-02):
+  - ``epsilon.amprouter`` added, providing support for multiplexing
+    unrelated *AMP* communications over the same connection.
+
+0.5.10 (2008-08-12):
+  - Added the ``epsilon.caseless`` module, with case-insensitive string
+    wrappers.
+  - Better ``repr()`` for ``epsilon.structlike.record`` added.
+  - ``epsilon.juice`` now uses ``twisted.internet.ssl`` instead of
+    ``epsilon.sslverify``.
+
+0.5.9 (2008-01-18):
+
+0.5.8 (2007-11-27):
+  - ``extime.Time.asHumanly`` no longer shows a time of day for all-day
+    timestamps.
+
+0.5.7 (2007-04-27):
+  - ``view.SlicedView`` added, allowing slicing and indexing of large
+    sequences without copying.
+
+0.5.6 (2006-11-20):
+  - Added a ``--quiet`` option to *Epsilon's* *certcreate* and use it in a few
+    unit tests to avoid spewing garbage during test runs.
+
+0.5.5 (2006-10-21):
+  - ``extime.Time`` now accepts RFC2822-like dates with invalid fields: it
+    rounds them to the nearest valid value.
+
+0.5.4 (2006-10-17):
+  - ``extime.Time`` now accepts RFC2822-like dates with no timezone.
+
+0.5.3 (2006-09-20):
+  - ``structlike.Record`` now raises ``TypeError`` on unexpected args.
+
+0.5.2 (2006-09-12):
+  - ``extime.Time`` now avoids ``time_t`` overflow bugs.
+
+0.5.1 (2006-06-22):
+  - Added hotfix for ``twisted.test.proto_helpers.StringTransport``.
+
+0.5.0 (2006-06-12):
+  - Replaced ``'%y'`` with ``'%Y'`` in ``Time.asHumanly`` output - the year is
+    now four digits, rather than two.
+  - Added new ``epsilon.structlike`` functionality for simple record.
+  - All uses of defer.wait and deferredResult were removed from the tests.
```

Comment:
Based on other examples, it seems like `defer.wait` and `deferredResult` should be wrapped in code markup here.